100_Numpy_exercises_with_hints_with_solutions.md

```diff
@@ -323,23 +324,22 @@ print(Z)
 `hint: np.add(out=), np.negative(out=), np.multiply(out=), np.divide(out=)`
 
 ```python
-A = np.ones(3)*1
-B = np.ones(3)*2
-C = np.ones(3)*3
+A = np.ones(3)
+B = np.full(3, 2)
```

Comment:
I prefer the version with ones (in order to keep it simple)

Reply:
Okay, I will rebase and drop commit. Should the `C` matrix stay as well?

Reply:
ok